app/src/main/java/vn/edu/usth/minigh/PullRequestFragment.java

```diff
@@ -0,0 +1,145 @@
+                    LinearLayout.LayoutParams.MATCH_PARENT,
+                    LinearLayout.LayoutParams.MATCH_PARENT
+            );
+            param.setMargins(16, 9, 9, 16);
+            param.gravity = Gravity.CENTER_HORIZONTAL;
+            ll.setOrientation(LinearLayout.HORIZONTAL);
+            ll.setClickable(true);
+            ll.setPadding(16, 16, 16, 16);
+            ll.setOnClickListener(new View.OnClickListener() {
+                @Override
+                public void onClick(View v) {
+                    Intent intent = new Intent(getActivity(), IssueDiscussionActivity.class);
+                    intent.setFlags(Intent.FLAG_ACTIVITY_NEW_TASK);
+                    startActivity(intent);
+                }
+            });
+            ll.setBackground(getResources().getDrawable(R.drawable.round_view_2nd));
+            ll.setLayoutParams(param);
+
+            ImageView img = new ImageView(this.getContext());
+            LinearLayout.LayoutParams imgParam = new LinearLayout.LayoutParams
+                    (ViewGroup.LayoutParams.WRAP_CONTENT, ViewGroup.LayoutParams.WRAP_CONTENT);
+            if(nStatus == "Close"){
+                img.setBackground(getResources().getDrawable(R.drawable.done));
+            } else {
+                img.setBackground(getResources().getDrawable(R.drawable.pricon));
+            }
+            img.setScaleType(ImageView.ScaleType.FIT_XY);
+            img.setPadding(0, 24, 0, 24);
+            imgParam.setMargins(16, 9, 0, 9);
+            img.setLayoutParams(imgParam);
+            ll.addView(img);
+
+            LinearLayout llchild = new LinearLayout(this.getContext());
+            LinearLayout.LayoutParams paramChild = new LinearLayout.LayoutParams(
+                    LinearLayout.LayoutParams.MATCH_PARENT,
+                    LinearLayout.LayoutParams.WRAP_CONTENT
+            );
+            paramChild.setMargins(16, 9, 9, 9);
+            llchild.setLayoutParams(paramChild);
+            llchild.setOrientation(LinearLayout.VERTICAL);
+
+            TextView reponame = new TextView(this.getContext());
+            LinearLayout.LayoutParams paramTxt = new LinearLayout.LayoutParams(
+                    LinearLayout.LayoutParams.WRAP_CONTENT,
+                    LinearLayout.LayoutParams.WRAP_CONTENT
+            );
+            reponame.setLayoutParams(paramTxt);
+            reponame.setTextSize(TypedValue.COMPLEX_UNIT_SP, 18f);
+            reponame.setText(ghname[i]);
+            llchild.addView(reponame);
+
+            TextView issueTxt = new TextView(this.getContext());
+            issueTxt.setLayoutParams(paramTxt);
+            issueTxt.setTextSize(TypedValue.COMPLEX_UNIT_SP, 18f);
+            issueTxt.setText(prList[i] +" "+ nStatus);
+            issueTxt.setTextColor(getResources().getColor(R.color.secondaryTextColor));
+            llchild.addView(issueTxt);
+
+            ll.addView(llchild);
+            check.addView(ll);
```

Comment:
I'd phrase it as defining this in XML and inflating it and edit the values that needs to be set dynamically.